src/bot.ts

```diff
@@ -495,6 +498,25 @@ a known bot info object.",
         }
     }
 
+    /**
+     * Returns true if the bot is currently running via built-in long polling,
+     * and false otherwise.
+     *
+     * If this method returns true, it means that `bot.start()` has been called,
+     * and that the bot has neither crashed nor was it stopped via a call to
+     * `bot.stop()`. This also means that you cannot use this method to check if
+     * a webhook server is running, or if grammY runner was started.
+     *
+     * Note that this method will already begin to return true even before the
+     * call to `bot.start()` has completed its initialization phase. During that
+     * period, it is `bot.isRunning() && !bot.isInited()`. By extension, this
```

Comment:
> During that period, it is `bot.isRunning() && !bot.isInited()`

I don't get what this is trying to tell me. Does it mean the function returns something else? Or that I should do something else. I think could be a bit more clear here 🤔

Reply:
It just means that just because the bot is running, does not mean it is inited. I will clarify this.